src/gui/attributetable/qgsdualview.cpp

```diff
@@ -118,11 +118,11 @@ void QgsDualView::init( QgsVectorLayer *layer, QgsMapCanvas *mapCanvas, const Qg
   mLayer = layer;
   mEditorContext = context;
 
-  connect( mTableView, &QgsAttributeTableView::willShowContextMenu, this, &QgsDualView::viewWillShowContextMenu );
+  connect( mTableView, &QgsAttributeTableView::willShowContextMenu, this, &QgsDualView::viewWillShowContextMenu, Qt::UniqueConnection );
```

Comment:
Wouldn't be better to make the connect in the constructor?

Reply:
I'll check it.

Reply:
I moved the connections for `mTableView` and `mFeatureListView` to the constructor. Can you have a look again? Thanks.